gns3fy/api.py

```diff
@@ -805,6 +806,52 @@ def nodes_summary(self, is_print=True):
 
         return _nodes_summary if not is_print else None
 
+    def nodes_inventory(self, is_print=True):
+        """return an ansible-type inventory with the nodes of the project for initial Telnet
+        configuration
+
+        e.g  
+    {  "spine00.cmh": {
+        "hostname": "127.0.0.1",
+        "username": "vagrant",
+        "password": "vagrant",
+        "port": 12444,
+        "platform": "eos",
+        "groups": [
+          "cmh"
+        ],
+      },
+      "spine01.cmh": {
+        "hostname": "127.0.0.1",
+        "username": "vagrant"
+        "password": "",
+        "platform": "junos",
+        "port": 12204,
+        "groups": [
+          "cmh"
+        ]
+      }
+    } 
+        """
+
+        if not self.nodes:
+            self.get_nodes()
+
+        _nodes_inventory = {}
+        _hostname = urlparse(self.connector.base_url).hostname
```

Comment:
I wonder what this `urlparse` is doing here?

Reply:
Urlparser is getting the GNS3 server IP address form the base_url on the GNS3 connector. ( as ansible should do telnet on that IP addreess and the port), possibly the IP address is natted or Whatever

I thought to create an attribute on the GNs3connector, but you can  can modify and re-factor is you wish

Reply:
Ah got it, cool. Yeah I was on the verge here trying to see if it makes sense to separate the IP address and port to use when connecting the GNS3 server... But since there are other stuff that I haven't tested like GNS3 servers on different computes, or servers using authentication I preferred to leave it like that. 

So yeah, is better for now the way you implemented it. We can perform some tests later and see if it makes sense to refactor